### DISCUSSION TITLE
Support for more than one set of RuxTabs on a webpage

### DISCUSSION THREAD

**Discussion by JoshWillisSynaptech** [RocketCommunicationsInc/astro#705] [Q&A] [answered]
I would like to use RuxTabs for more than one component on my webpage. When a new tab is selected on one set of RuxTabs it changes the content of all the RuxTabPanels on the webpage because the ruxselected event is the trigger for tab change. 

Is there a supported method to use more than one set of tabs on a page and have the content only of the specific tab group update when a tab is selected?

**Comment by micahjones13** [comment]
Hey @JoshWillisSynaptech, thanks for the discussion! 
Just to clarify here, is the issue you're running into that when one set of tabs get's clicked, the other set stops displaying what it had been displaying? For example, if you check out [this codesandbox](https://codesandbox.io/s/cool-bird-ordxg6?file=/index.html), when you click one set of tabs the tab-panel on the other set hides itself. Just want to make sure I'm looking at the right thing

**Reply by JoshWillisSynaptech** [reply]
Thanks @micahjones13,
Yes this is exactly the issue I am running into.

**Reply by micahjones13** [reply]
Sweet! I'm going to create a bug issue from this discussion then and start working on it today. I'll keep you posted!

**Comment by micahjones13** [comment, marked as answer]
Alright, the PR to fix this issue has been merged and will be released on 9/15. Thanks for raising this, and let us know if anything else comes up! 
Link to bug fix PR: https://github.com/RocketCommunicationsInc/astro/pull/710

**Reply by JoshWillisSynaptech** [reply]
Thank you for fixing this issue!
